src/colinajose/service/SchoolService.java

```diff
@@ -46,12 +49,16 @@ public String registerCourse(String grade, String year, String group){
     public String registerParent(String name, String ci, int age, String gender, String address, String phone, String email){
         Contact contact = new Contact(address, phone, email);
         Parent parent = new Parent(name, ci, age, gender, contact);
-        school.addParent(parent);
+        Device device =  new Device(parent);
```

Comment:
Is not mandatory for the parents to have a Device. So registerDevice should be another action available.

Reply:
@JurgenPadilla fixed in next commit